next.config.js

```diff
@@ -337,6 +337,31 @@ async function redirects() {
       permanent: true,
```

Comment:
@agraebe can you go through and ensure that the old redirects are updated to? such as https://github.com/blockstack/docs/blob/feat/stacks-integration-guide/next.config.js#L226-L228

Reply:
jup, done!